sql/core/pom.xml

```diff
@@ -277,22 +277,22 @@
           <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
           <artifactSet>
             <includes>
-              <include>com.google.protobuf:*</include>
+              <include>org.spark-project.spark:unused</include>
             </includes>
           </artifactSet>
           <relocations>
             <relocation>
-              <pattern>com.google</pattern>
-              <shadedPattern>${spark.shade.packageName}.com.google</shadedPattern>
-              <excludes>
-                <!-- Guava is relocated to ${spark.shade.packageName}.connect.guava -->
-                <exclude>com.google.common.**</exclude>
-              </excludes>
+              <pattern>com.google.common</pattern>
```

Comment:
cc @pan3793

Reply:
the effective relocation rules are very confusing, seems to be affected by parent pom

Reply:
We can use `combine.self = "override"` to resolve this issue.